module/documents/actor.mjs

```diff
@@ -165,12 +246,16 @@ export default class ActorEd extends Actor {
       return;
     }
     // Check if this uses karma or strain at all
-    this.takeDamage( roll.edRollOptions.strain, "standard" );
+    this.takeDamage( roll.options.strain.total, "standard", undefined, true );
     if (
-        !this.#useResource( 'karma', roll.edRollOptions.karma.pointsUsed )
-        || !this.#useResource( 'devotion', roll.edRollOptions.devotion.pointsUsed )
+        !this.#useResource( 'karma', roll.options.karma.pointsUsed )
+        || !this.#useResource( 'devotion', roll.options.devotion.pointsUsed )
     ) {
-      ui.notifications.warn("Localize: Not enough karma or devotion. Used all that was available.");
+      ui.notifications.warn( "Localize: Not enough karma or devotion. Used all that was available." );
```

Comment:
These checks should be done before prompting the roll. It's pretty frustrating to only know something isn't possible after you prepared everything for it.
Maybe we need something like `#preProcessRoll` method

Reply:
Oh i think you did taht acutally, I just added a space for lint sake :-P